packages/api/src/beacon/routes/beacon/rewards.ts

```diff
@@ -0,0 +1,88 @@
+import {ContainerType} from "@chainsafe/ssz";
+import {ssz, ValidatorIndex} from "@lodestar/types";
+
+import {
+  RoutesData,
+  ReturnTypes,
+  Schema,
+  ReqSerializers,
+  ContainerDataExecutionOptimistic,
+} from "../../../utils/index.js";
+import {HttpStatusCode} from "../../../utils/client/httpStatusCode.js";
+import {ApiClientResponse} from "../../../interfaces.js";
+import {BlockId} from "./block.js";
+
+// See /packages/api/src/routes/index.ts for reasoning and instructions to add new routes
+
+/**
+ * True if the response references an unverified execution payload. Optimistic information may be invalidated at
+ * a later time. If the field is not present, assume the False value.
+ */
+export type ExecutionOptimistic = boolean;
+
+export type ProposerRewardsResponse = {
+  proposerIndex: ValidatorIndex;
+  total: number;
+  attestations: number;
+  syncAggregate: number;
+  proposerSlashings: number;
+  attesterSlashings: number;
+};
+
+export type Api = {
+  /**
+   * Get block rewards
+   * Returns the info of rewards received by the block proposer
+   *
+   * @param blockId Block identifier.
+   * Can be one of: "head" (canonical head in node's view), "genesis", "finalized", \<slot\>, \<hex encoded blockRoot with 0x prefix\>.
+   */
+  getProposerRewards(
```

Comment:
The operationId does not match with the one [defined in the beacon api spec](https://github.com/ethereum/beacon-APIs/blob/c458197bb5529b2169fe51888f66f3581380b94f/apis/beacon/rewards/blocks.yaml#L2)

```suggestion
  getBlockRewards(
```

See reasoning of the API definitions
https://github.com/ChainSafe/lodestar/blob/551fd8e4553d07c835e31e2b7939ab5f18c5f95f/packages/api/src/beacon/routes/index.ts#L33-L34